### PR TITLE
Bump up versions to latest

### PR DESCRIPTION
**Pull request by Cotopax**
Would be nice to have this library with the latest dependencies and scala version

### CODE REVIEW

**Comment by matiwinnetou** [comment]
The ideal solution would be something that I did in https://github.com/matiwinnetou/swagger-play24

It just need a copy and paste from there to this project.

There crossVersions are supported and with one command I can cross release this for many scala versions.

File: README.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ Number of notable differences comparing to Netflix Hystrix:
...
 
-- Play for Java 2.3.x and binary available for Scala 2.10.x (cross compilation for Scala 2.11.x binary on request).
+- Play for Java 2.4.x and binary available for Scala 2.11.x
 - Java 8
 
 ## Status 
 status: used in production, interface stable, only additions planned
 
-http://repo1.maven.org/maven2/pl/matisoft/play-hysterix_2.10/
+http://repo1.maven.org/maven2/pl/matisoft/play-hysterix_2.12/
```
Comment (matiwinnetou) [inline README.md]: this actually is not correct as 2.10 refers to a scala version not hysterix.

File: release.sbt [hunk 2]
```diff
@@ -1,35 +1,24 @@
-import sbtrelease._
-import ReleaseStateTransformations._
-import ReleasePlugin._
-import ReleaseKeys._
+import ReleaseTransformations._
```
Comment (Cotopax) [inline release.sbt]: Release sbt from swagger-play24

File: project/plugins.sbt [hunk 3]
```diff
@@ -5,12 +5,10 @@ logLevel := Level.Warn
 resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/"
 
 // Use the Play sbt plugin for Play projects
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.3.8")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.4.6")
```
Comment (Cotopax) [inline project/plugins.sbt]: Also copy from swagger-play24 but use latest play sbt-plugin 2.4.6 instead of 2.4.0

**Pull request merged**